src/mainboard/amd/romecrb/src/main.rs

```diff
@@ -201,7 +215,10 @@ pub extern "C" fn _start(fdt_address: usize) -> ! {
     uart0.pwrite(b"Welcome to oreboot\r\n", 0).unwrap();
     debug.init().unwrap();
     debug.pwrite(b"Welcome to oreboot - debug port 80\r\n", 0).unwrap();
-    let s = &mut [debug as &mut dyn Driver, uart0 as &mut dyn Driver];
+    let p0 = &mut AMDMMIO::new(0xfedc9000);
```

Comment:
Comment magic number or set as a const at the top?

Reply:
I am halfway thinking of allowing an Option parameter and if it is not passed having that default to COM1 as known to the driver, wdyt? 99.99999% of the uses will be for COM1, so why make them have to pass the parameter.

Reply:
better yet: AMDMMIO:com1()
then it's clear what's up